### PR TITLE
Implement Organizer UI : Add Event

### PR DESCRIPTION
**Pull request by AlexisTabin**
OrganizerUI is almost the same as AttendeeUI with just a few differences : 

1. For Future Events category, the organizer can click on a "+" Button to add events
2. When the "+" Button is clicked, it opens an alert dialog with which the organizer selects the event type

- [x] Add "+" Button for Future Events
- [x] Click on "+" open an Alert Dialog to choose which event Organizer wants to create

### CODE REVIEW

File: app/src/main/java/com/github/dedis/student20_pop/ui/OrganizerFragment.java [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+                new Keys().getPublicKey(), "Lausanne", "Discussion"));
+        events.add(new Event("Future Event 1", new Date(5000 * 1000L),
+                new Keys().getPublicKey(), "i don't know where yet", "Poll"));
+
+        return events;
+    }
+
+    public void launchEventCreationFragment(int eventType) {
+        switch (eventType) {
+            case 0:
```
Comment (celinecamacho) [inline app/src/main/java/com/github/dedis/student20_pop/ui/OrganizerFragment.java:109]: Is it possible to have an enum with the types? How do we know that 0 is for Meeting Event, 1 is for Roll-Call, and so on?

File: app/src/main/res/drawable/ic_baseline_add_24.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
```
Comment (celinecamacho) [inline app/src/main/res/drawable/ic_baseline_add_24.xml:5]: These values should be in the dimens.xml file

File: app/src/main/res/drawable/ic_baseline_edit_24.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
```
Comment (celinecamacho) [inline app/src/main/res/drawable/ic_baseline_edit_24.xml:5]: These values should also be in the dimens.xml file

File: app/src/main/res/drawable/ic_baseline_add_24.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    android:tint="?attr/colorControlNormal">
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M19,13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"/>
```
Comment (celinecamacho) [inline app/src/main/res/drawable/ic_baseline_add_24.xml:9]: Can you save this value in the strings.xml file?

File: app/src/main/res/drawable/ic_baseline_edit_24.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    android:tint="?attr/colorControlNormal">
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M3,17.25V21h3.75L17.81,9.94l-3.75,-3.75L3,17.25zM20.71,7.04c0.39,-0.39 0.39,-1.02 0,-1.41l-2.34,-2.34c-0.39,-0.39 -1.02,-0.39 -1.41,0l-1.83,1.83 3.75,3.75 1.83,-1.83z"/>
```
Comment (celinecamacho) [inline app/src/main/res/drawable/ic_baseline_edit_24.xml:9]: Can you save this value in the strings.xml file?

File: app/src/main/res/layout/event_category_layout.xml [hunk 6]
```diff
@@ -1,18 +1,34 @@
...
     android:layout_height="match_parent"
-    android:background="@color/categoryTab">
+    android:background="@color/categoryTab"
+    android:gravity="center_vertical">
 
     <TextView
         android:id="@+id/event_category"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_marginTop="5dp"
```
Comment (celinecamacho) [inline app/src/main/res/layout/event_category_layout.xml:14]: This value should be in the dimens.xml file

File: app/src/main/res/layout/properties_layout.xml [hunk 7]
```diff
@@ -1,25 +1,29 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
     <TextView
         android:id="@+id/organization_name"
-        android:layout_width="match_parent"
+        android:layout_width="350dp"
```
Comment (celinecamacho) [inline app/src/main/res/layout/properties_layout.xml:10]: Why did you modify the layout width? It should match the parent layout

**Review by celinecamacho — CHANGES_REQUESTED** [review]
Good, just some conventions that should be followed

File: app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    @Test
+    public void canLaunchCreateMeetingEventFragment(){
+        onView(allOf(withId(R.id.add_future_event_button),
+                withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
+                .perform(click());
+        onView(withText(getApplicationContext().getString(R.string.meeting_event))).check(matches(isDisplayed()));
+        onView(withText(getApplicationContext().getString(R.string.meeting_event))).perform(click());
+        //TODO : Check that the corresponding Fragment has been launched
```
[the remark this answers is not in the document]
Comment (AlexisTabin) [reply app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java:68]: Do you want me to remove them or to add the @Ignore tag  ?

File: app/src/main/res/layout/properties_layout.xml (hunk 7 above)
Comment (AlexisTabin) [reply app/src/main/res/layout/properties_layout.xml]: corrected

File: app/src/main/res/layout/event_category_layout.xml (hunk 6 above)
Comment (AlexisTabin) [reply app/src/main/res/layout/event_category_layout.xml]: done

File: app/src/main/res/drawable/ic_baseline_edit_24.xml (hunk 5 above)
Comment (AlexisTabin) [reply app/src/main/res/drawable/ic_baseline_edit_24.xml]: sure

File: app/src/main/res/drawable/ic_baseline_add_24.xml (hunk 4 above)
Comment (AlexisTabin) [reply app/src/main/res/drawable/ic_baseline_add_24.xml]: sure

File: app/src/main/res/drawable/ic_baseline_edit_24.xml (hunk 3 above)
Comment (AlexisTabin) [reply app/src/main/res/drawable/ic_baseline_edit_24.xml]: done

File: app/src/main/res/drawable/ic_baseline_add_24.xml (hunk 2 above)
Comment (AlexisTabin) [reply app/src/main/res/drawable/ic_baseline_add_24.xml]: done

File: app/src/main/java/com/github/dedis/student20_pop/ui/OrganizerFragment.java (hunk 1 above)
Comment (AlexisTabin) [reply app/src/main/java/com/github/dedis/student20_pop/ui/OrganizerFragment.java]: I tried something, tell me I you prefer it that way

File: app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java (hunk 8 above)
[the remark this answers is not in the document]
Comment (AlexisTabin) [reply app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java]: I added a @Ignore Tag

**Review by celinecamacho — APPROVED** [review]

File: app/src/main/java/com/github/dedis/student20_pop/ui/OrganizerFragment.java (hunk 1 above)
Comment (celinecamacho) [reply app/src/main/java/com/github/dedis/student20_pop/ui/OrganizerFragment.java]: It's better, thanks!

**Review by jbmichel14 — APPROVED** [review]
All good!

**Pull request merged**